### PR TITLE
Feature/connect postgre sql

### PR DESCRIPTION
**Pull request by SerjiKSM**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    username: admin_wst
+    password: wst!#4pas@*
+    url: jdbc:postgresql://localhost:5432/db_wst
+    platform: postgres
+    initialization-mode: always
+  jpa:
+    show-sql: true
+    hibernate.ddl-auto: none
```
Comment (Sabfir) [inline src/main/resources/application.yml]: Plz change this to
```
hibernate:
  ddl-auto: none
```

**Review by Sabfir — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+    database-platform: org.hibernate.dialect.PostgreSQL94Dialect
+    properties:
+      hibernate:
+        temp:
+          use_jdbc_metadata_defaults: false
```
Comment (Sabfir) [inline src/main/resources/application.yml]: What is this property? where did you get this example?

**Comment by Sabfir** [comment]
I´d like to go with minimal settings if we are not sure we need others.
Can you check if this set of properties is enough?
```
    datasource:
        url: <url>
        username: <username>
        password: <password>
    jpa:
        generate-ddl: false
        hibernate:
            ddl-auto: none
        show-sql: true
```

File: src/main/resources/application.yml (hunk 1 above)
Comment (SerjiKSM) [reply src/main/resources/application.yml:13]: https://o7planning.org/ru/11897/spring-boot-and-spring-data-jpa-tutorial
# Fix Postgres JPA Error:
# Method org.postgresql.jdbc.PgConnection.createClob() is not yet implemented.
spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false

File: src/main/resources/application.yml (hunk 2 above)
Comment (SerjiKSM) [reply src/main/resources/application.yml]: https://o7planning.org/ru/11897/spring-boot-and-spring-data-jpa-tutorial

# Fix Postgres JPA Error:
# Method org.postgresql.jdbc.PgConnection.createClob() is not yet implemented.
spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false

**Review by Sabfir — APPROVED** [review]

**Pull request merged**